### PR TITLE
mgr/dashboard: run-backend-api-tests.sh: Older setuptools

### PR DESCRIPTION
**Pull request by djgalloway**
https://github.com/pypa/setuptools/issues/2353

Fixes: https://tracker.ceph.com/issues/53675

Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

**Comment by alfonsomthd** [comment]
jenkins test windows

File: src/pybind/mgr/dashboard/run-backend-api-tests.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ setup_teuthology() {
 
     ${TEUTHOLOGY_PYTHON_BIN:-/usr/bin/python3} -m venv venv
     source venv/bin/activate
-    pip install -U pip 'setuptools >= 12'
+    pip install -U pip 'setuptools>=12,<60'
```
Comment (epuertat) [inline src/pybind/mgr/dashboard/run-backend-api-tests.sh:46]: Good catch @djgalloway ! Thanks a lot... I was an old fan of rolling deps, but I'm becoming more and more conservative when it comes to leaving things run loose 🙈

**Review by epuertat — APPROVED** [review]

**Review by alfonsomthd — APPROVED** [review]
@djgalloway Thanks for the hotfix!

**Comment by ljflores** [comment]
The api and windows test failures are because of braggi12.

**Comment by alfonsomthd** [comment]
> The api and windows test failures are because of braggi12.

It seems so: I see:
```
mkdir: cannot create directory '/tmp/install-deps.1741570': No space left on device
```

**Comment by ljflores** [comment]
jenkins test make check

**Comment by djgalloway** [comment]
jenkins retest

**Comment by ljflores** [comment]
jenkins retest this please

**Review by ljflores — APPROVED** [review]
@djgalloway all checks passed -- looks good!

**Pull request merged**